### DISCUSSION TITLE
Offline installation on Visual Studio 2022

### DISCUSSION THREAD

**Discussion by ischuldt** [wixtoolset/issues#7252] [Questions] [answered]
I have installed Visual Studio Enterprise 2022 on an **offline** machine. When I attempt to run the Votive2022.vsix extension it fails saying there was a bad timestamp. I've seen similar questions about VS 2019, and the only answer given is that you have to export some certificates from a machine that has it installed properly and manually import them onto the offline machine, but it doesn't specifically say what Certificates I need to import, and how exactly I should do that.

I tried exporting some dotnet and digiCert certificates using Chrome, and then imported them using Chrome on the offline machine, but that didn't work. So there must be something else I'm missing or the import was wrong. 

The certs I tried were: .net Foundation Code Signing CA2 and a DigiCert High Assurance EV Root.

What else am I missing?

**Comment by BMurri** [comment, marked as answer]
Possibly something like [this](https://woshub.com/updating-trusted-root-certificates-in-windows-10/)?

**Reply by ischuldt** [reply]
This did eventually work. I had to install literally all of the certs from my computer on the other one. I'm not sure which one actually did it. Even after doing that, it still gave me a bad timestamp message, but when I ran the Votive2022.vsix -force.
